### PR TITLE
docs(example-plugins): adding nvim-neo-tree alternative to nvim-tree

### PR DESCRIPTION
**Pull request by gacallea**
[CHECKLIST]
 - [x] I have read the [contributing guidelines](https://github.com/LunarVim/lunarvim.org/blob/master/CONTRIBUTING.md)
 - [x] I prefixed the title with docs: on documentation updates

neo-tree is an amazing alternative to nvim-tree. been using it for a long while and never had issues. it also offers a wider feature set and works flawlessly. I used the stable branch in my config, as neo-tree cares about that to provide solid software.

hope this helps.

### CODE REVIEW

File: versioned_docs/version-1.2/plugins/extra-plugins.md [hunk 1]
```diff
@@ -168,6 +168,55 @@ lvim.autocommands = {
...
+        },
+      },
+    })
+  end
+},
+```
+
+After installing ensure to disable `nvim-tree` it in your `config.lua` using:
+
+```
```
Comment (LostNeophyte) [inline versioned_docs/version-1.2/plugins/extra-plugins.md:214]:
```suggestion
```lua
```

File: docs/configuration/plugins/example-configurations.md [hunk 2]
```diff
@@ -132,6 +132,55 @@ For a more lightweight, easier-to-use alternative, check out the author's new, w
...
+        },
+      },
+    })
+  end
+},
+```
+
+After installing ensure to disable `nvim-tree` it in your `config.lua` using:
+
+```
```
Comment (LostNeophyte) [inline docs/configuration/plugins/example-configurations.md:178]:
```suggestion
```lua
```

File: docs/configuration/plugins/example-configurations.md [hunk 3]
```diff
@@ -132,6 +132,55 @@ For a more lightweight, easier-to-use alternative, check out the author's new, w
...
 
+### [neo-tree](https://github.com/nvim-neo-tree/neo-tree.nvim)
+
+**neo-tree is a plugin to manage the file system and other tree like structures**
+
+```lua
+{
+  "nvim-neo-tree/neo-tree.nvim",
+  branch = "v2.x",
+  requires = {
```
Comment (LostNeophyte) [inline docs/configuration/plugins/example-configurations.md:143]:
```suggestion
  dependencies = {
```
we use lazy instead of packer on master

**Comment by LostNeophyte** [comment]
Thanks!

**Comment by gacallea** [comment]
that was fast! thank you @LostNeophyte ❤️

**Pull request merged**